website_and_docs/config.toml

```diff
@@ -8,7 +8,7 @@ theme = ["docsy"]
 
 # Will give values to .Lastmod etc.
 enableGitInfo = true
-
+#ignoreErrors = ["error-remote-getjson"] 
```

Comment:
This might not be needed at all. Can we revert it as it was before!

Reply:
this is done.

Reply:
changed.